### DISCUSSION TITLE
export line to GPX

### DISCUSSION THREAD

**Discussion by animaluminium** [OpenOrienteering/mapper#2134] [Ideas]
I'd like to export a OOM line to GPX, so I can follow it with my sports watch. Like this, I can use the watch to navigate between mapping spots and open the app only at mapping spots. Also, it could help me create routes for training but that's secondary.

**Comment by kjetilk** [comment]
I would have a use for this too. I'm organising ski-o event, and as part of that, we will have a tracks that are currently commonly in use, but has not yet been adequately mapped, and so I'll map them on a georeferenced map. 

There's a local membership association that grooms more than a thousand kms of tracks asked us if we could help them with accurate GPX files of those trails, and we would be happy to contribute that, and the most accurate we'd have would be a line export from the map I'm currently making.

**Reply by Choreuomai** [reply]
But gpx export is possible? Or did you mean that you specifically want to export one particular object?
By the way, for your ski tracks, have you considered adding them directly to OpenStreetMap? Most of the time, objects from orienteering maps are not so interesting for the general public so I don't add them to OSM, but that is different with ski tracks in my opinion.

**Comment by kjetilk** [comment]
> But gpx export is possible? Or did you mean that you specifically want to export one particular object? 

Yeah, or rather a collection of objects. Actually, I have them in a part.

> By the way, for your ski tracks, have you considered adding them directly to OpenStreetMap? 

Actually, that's a really good point! In fact, they are already there, so I could just point them to that.
